label_sync/labels.yaml

```diff
@@ -663,6 +663,11 @@ default:
       target: both
       prowPlugin: milestonestatus
       addedBy: "members of a configurable github team. default: [@kubernetes/kubernetes-milestone-maintainers](https://github.com/orgs/kubernetes/teams/kubernetes-milestone-maintainers/members)"
+    - color: ffaa00
+      description: Denotes a PR that should be squashed by tide when it merges.
+      name: tide/squash
+      target: prs
+      addedBy: humans
```

Comment:
I think human added is fine to start with, but we'll want to change this eventually. Perhaps once I have time to finish the command framework?

Reply:
yeah, that was my thought too. I think I would want to have this be maintainer-ish restricted, rather than open to anyone

Reply:
I'm mildly wary of `tide/` as the prefix, what about something boring like `merge-method/squash`

Reply:
The concern I had with that is it would be a "class" of one label. I could see "tide/" being used for other labels, like merge blockers, or no retest.